Packs/Qintel/Playbooks/playbook-Indicator_Enrichment_-_Qintel.yml

```diff
@@ -461,12 +463,6 @@ inputs:
   required: false
   description: IPs to be enriched
   playbookInputQuery:
-- key: Domain
```

Comment:
this wasnt used at all? Lets track down what the reason it was added was

Reply:
wasn't used at all, even got a validation failed on it.
Hard to track down because it was a contribution.